src/OperatingSystem.ts

```diff
@@ -0,0 +1,12 @@
+import { type } from './decorators'
+import Intangible from './Intangible'
+
+@type('schema:OperatingSystem')
+export default class OperatingSystem extends Intangible {
+
+  // Instances of Operating Systems (high level)
+  static linux: OperatingSystem = new OperatingSystem({ name: 'Linux' })
```

Comment:
```suggestion
  // See http://tinyurl.com/y9u34osp for names of operating systems in Wikidata
  static linux: OperatingSystem = new OperatingSystem({ name: 'Linux' })
```

Reply:
I just added this note because `macOS` caught my eye as a being a different capitalisation from the others. But Wikidata suggests this is correct :+1: I'm wondering if we should use Wikidata in this way when we want to create instances of types e.g. for `ComputerLanguage` as well